app/sim_bench_test.go

```diff
@@ -41,8 +40,7 @@ func BenchmarkFullAppSimulation(b *testing.B) {
 	app := New(logger, db, nil, true, map[int64]bool{}, b.TempDir(), simcli.FlagPeriodValue, MakeEncodingConfig(), simtestutil.EmptyAppOptions{}, interBlockCacheOpt())
 
 	// run randomized simulation
-	// TODO[1760]: event-history: Add _ return arg back in.
-	_, simParams, simErr := simulation.SimulateFromSeed(
+	_, _, simParams, simErr := simulation.SimulateFromSeed(
```

Comment:
The `SimulateFromSeed` function still returns only three arguments. I made `SimulateFromSeedProv` that also returns the last block time of the simulation. So for the tests where we don't need it (like this one) we can still use `SimulateFromSeed`.

Reply:
Weird, I edited the vendor simulate file to return the block time for `SimulateFromSeed` so everything was compiling locally (not realizing it was in the comos-sdk).  It used to not allow that and wouldn't compile.  Not sure what changed... Anyway, makes sense and is fixed now.